posts/2021-06-21-an-intent-to-preload.md

```diff
@@ -0,0 +1,50 @@
+---
+title: An intent to preload
+description: Governments should be easy to identify on the internet and users should be secure on .gov websites.
+layout: layouts/post
+date: 2021-06-21
+author: Cameron Dixon
+excerpt: Governments should be easy to identify on the internet and users should be secure on .gov websites. Today, the .gov program announces our intent to preload the .gov top-level domain.
+tags:
+  - posts
+  - Program updates
+  - Security
+---
+
+When the public sees information on a .gov website, they are likely to trust that it is official and accurate. This trust is warranted, since registration of a .gov domain is limited to bona fide U.S.-based government organizations. Governments should be easy to identify on the internet and users should be secure on .gov websites.
+
+HTTPS is a key protection for websites and web users. It offers security and privacy when connecting to the web, and provides governments the assurance that what they publish is what is delivered to users. In the last few years, HTTPS has become the default connection type on the web. Browsers that were once telling users to “watch for a green lock!” are now removing the lock icons. Instead, the browser warns users when sites are not using HTTPS.
+
+<!--
+Commenting out this image for the time being.
+{% image "_img/securedotgov.png" "Governments should never be not secure" %}
+
+Governments should never be "not secure"
+-->
+
```

Comment:
Same here, though also just wanted to check if there was a particular reason you wanted to comment this one out (mostly out of curiosity)

Reply:
I can see people getting the wrong impression, but preloading is a complicated topic and the image is a nice visual indication of what preloading would eliminate. Can we address potential confusion with better text near the image?